src/explore-education-statistics-frontend/src/modules/permalink/PermalinkPage.tsx

```diff
@@ -64,19 +64,19 @@ const PermalinkPage: NextPage<Props> = ({ data }) => {
       </div>
 
       {data.status === 'SubjectRemoved' && (
-        <WarningMessage error>
+        <WarningMessage error testId="permalink-warning">
```

Comment:
The tooltip that shows up if the table can't be rendered & the warning share the same className so I was a bit worried about false positives getting flagged because of this

Reply:
Not sure there'd be false positives, but happy to go with the testids anyway as they might be useful in the future!